test/nixl/nixl_test.cpp

```diff
@@ -204,7 +204,7 @@ static void initiatorThread(nixlAgent &agent, nixl_opt_args_t *extra_params,
 }
 
 static void runTarget(const std::string &ip, int port) {
-    nixlAgentConfig cfg(true, true, port, nixl_thread_sync_t::NIXL_THREAD_SYNC_STRICT);
```

Comment:
Should we test both?

Reply:
I don't see why - both models take exactly the same lock by exactly the same macro. And if tsan doesn't produce any errors with more permissive mode of that lock than it won't also with more strict.